client/components/tooltip/index.tsx

```diff
@@ -11,23 +11,31 @@ import TooltipBase, { TooltipBaseProps } from './tooltip-base';
 
 type TooltipProps = TooltipBaseProps & {
 	isVisible?: boolean;
+	ignoreMouseHover?: boolean;
 	onHide?: () => void;
 };
 
 const Tooltip: React.FC< TooltipProps > = ( {
 	isVisible,
+	ignoreMouseHover = false,
```

Comment:
> Can we make this parameter more semantic and/or positively worded?
`clickEventsOnly`? Another option is to make a separate component, e.g. `ClickableTooltip`. What do you think?

> Or can we infer that mode from the content passed in?
I think being more explicit about the behaviour is preferred. There are uses of this Tooltip component where a "Learn more" doc link exists within its content, e.g the settings screen:
<img width="321" alt="image" src="https://user-images.githubusercontent.com/3147296/227041918-dbda1c5a-5b88-459f-b4df-98b36b3f53b0.png">

> necessary for tooltips that contain links to docs.
If this "clickable" tooltip is to be required for all tooltips with docs links, I could move this to a new issue/PR to update the component and change the existing behaviour.

Reply:
> If this "clickable" tooltip is to be required for all tooltips with docs links, I could move this to a new issue/PR to update the component and change the existing behaviour.

Pretty sure that's the case (my assumption – might be clarification in design comments in figma or P2). You could confirm with @elizaan36 , here's how I understand it:

- Existing help icon + tooltip component shows a text message on mouseover/hover. The message auto-hides (i.e. mouse away / timeout).
- The ones we need look similar but behave differently, because the user needs to be able to click on the docs link. This behaviour should be used for any help icon+tooltip that has a link (or maybe for long messages, etc).

Reply:
> I think being more explicit about the behaviour is preferred.

I agree :) 

> ClickableTooltip

Sounds reasonable :)

Reply:
+1 on everything here. This is a new variant/behavior for the existing icon + tooltip to account for the docs link. It's also optimized for mobile.

Reply:
Thanks @elizaan36 and @haszari, I'll make it happen 👍

Reply:
I've created and assigned myself to a new issue #5851 for implementing the new click-only Tooltip variant and updating existing tooltips with doc links. I'll update this PR description to mention this.